docs/.vuepress/config.js

```diff
@@ -12,13 +12,16 @@ module.exports = {
   chainMarkdown (config) {
     config.plugin('add-metadata')
       .use(markdown => {
-        markdown.core.ruler.push('add-date', state => {
+        markdown.core.ruler.push('add-metadata', state => {
```

Comment:
I found it hard to find documentation for this bit. Maybe add a link to it for future developers?